src/main/java/com/minecolonies/coremod/colony/buildings/BuildingDeliveryman.java

```diff
@@ -20,7 +21,10 @@
 
     private static final String DELIVERYMAN = "Deliveryman";
 
-    private static final String TAG_DELIVERY = "delivery";
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)